node/src/Unison/Runtime/ExtraBuiltins.hs

```diff
@@ -37,17 +41,35 @@ makeRandomHash genVar = do
   _ <- MVar.swapMVar genVar newGen
   pure hash
 
+makeRandomBS :: RandomGen r => MVar.MVar r -> IO ByteString
+makeRandomBS genVar = Hash.toBytes <$> makeRandomHash genVar
+
+makeRandomSeries :: RandomGen r => MVar.MVar r -> IO Series
+makeRandomSeries genVar = Series <$> makeRandomBS genVar
+
+makeRandomAddress :: RandomGen r => MVar.MVar r -> IO Address.Address
+makeRandomAddress genVar = Address.Address <$> makeRandomBS genVar
+
+makeRandomIdentifier :: RandomGen r => MVar.MVar r -> IO (Series, Series)
+makeRandomIdentifier genVar = do
+  cp <- makeRandomSeries genVar
+  ud <- makeRandomSeries genVar
+  pure (cp, ud)
+
+
 makeAPI :: IO (WHNFEval -> [Builtin])
 makeAPI = do
   stdGen <- getStdGen
   genVar <- MVar.newMVar stdGen
-  resourcePool <- RP.make 3 10 KVS.load KVS.close
-  let nextHash = makeRandomHash genVar
+  let nextAddress = makeRandomAddress genVar
+      nextID = makeRandomIdentifier genVar
+  blockStore <- FBS.make' nextAddress "keyValueStore"
+  resourcePool <- RP.make 3 10 (KVS.load blockStore) (const (pure ()))
   pure (\whnf -> map (\(r, o, t, m) -> Builtin r o t m)
      [ let r = R.Builtin "KeyValue.empty"
            op [] = Note.lift $ do
-             hash <- nextHash
-             pure . store . Term.lit . Term.Text . Hash.base64 $ hash
+             ident <- nextID
+             pure . store . Term.lit . Term.Text . KVS.idToText $ ident
            op _ = fail "KeyValue.empty unpossible"
            type' = unsafeParseType "forall k v. Remote (Store k v)"
        in (r, Just (I.Primop 0 op), type', prefix "stringStore")
```

Comment:
`prefix "stringStore"` can just be `prefix "empty"`

Minor thing:

* Can we rename `KeyValueStore` to `Index`. So the type will be `Index k v`, and the various builtins will be `Index.empty`, `Index.insert`, etc. Use the convention that `Index.foo` has `prefix "foo"` as rendering. The editor will make it easy to resolve even if there are multiple things called `foo`.